### DISCUSSION TITLE
Context Menu Shortcut in Windows 11

### DISCUSSION THREAD

**Discussion by rohitxkv** [localsend/localsend#1871] [Discussion (English, International)]
**Context Menu Shortcut** in Windows 11 New Menu (For folders & Files Selections)
![Context Menu](https://github.com/user-attachments/assets/122ad857-6687-42c5-870f-3fc69d066a3c)

**Windows Share Button Shortcut**
![Windows Share Button](https://github.com/user-attachments/assets/dc80e2b6-3226-4ecb-b7ba-8e9918f0d5ff)

**Comment by PedanXr** [comment]
This would be so helpfull!
